### ISSUE TITLE
Native video controls are not accessible with screen readers ( tested with chrome)

### ISSUE THREAD

**Issue by padms** [equinor/energyvision#2209]

**Comment by padms** [comment]
@meols We are using Videojs player . The player now uses custom controls from videojs instead of the native controls, which has accessible names for the buttons.

**Comment by meols** [comment]
@padms - can you please specify what I should test?

**Comment by padms** [comment]
@meols The existing video player does not read the controls on the video loud with the screen reader, the new player supports accessibility for the controls.

**Comment by meols** [comment]
@padms - this is approved :thumbsup:
